recipe-server/client/control_new/state/app/recipes/reducers.js

```diff
@@ -67,11 +68,23 @@ function items(state = new Map(), action) {
 
 function listing(state = new Map(), action) {
   switch (action.type) {
-    case RECIPE_PAGE_RECEIVE:
+    case RECIPE_PAGE_RECEIVE: {
+      let totalRecipes = state.get('all', new Set());
+
+      fromJS(action.recipes.results.forEach(result => {
+        totalRecipes = totalRecipes.add(fromJS(result));
+      }));
+
+      totalRecipes = totalRecipes.sort((a, b) => a.get('id') <= b.get('id'));
```

Comment:
This isn't correct. The comparator function passed to `.sort()` [should return -1, 0, or 1](https://facebook.github.io/immutable-js/docs/#/Set/sort), not a boolean. Assuming `id` is a number, `a.get('id') - b.get('id')` would would.